spec/requests/likes_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "Likes", type: :request do
+  describe "GET /index" do
```

Comment:
please add proper specs

Reply:
I will delete the file because it is not used.